oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnectionRule.java

```diff
@@ -154,17 +203,45 @@ public ElasticConnection getElasticConnectionFromString() {
         }
     }
 
+    private boolean isValidUri(String connectionString) {
+        if (connectionString == null) {
```

Comment:
I wonder if we should use "connectionString == null || connectionString.trim().isEmpty()"

Reply:
second condition is handled by URI instantiating